packages/transaction-controller/src/types.ts

```diff
@@ -132,6 +132,9 @@ type TransactionMetaBase = {
    */
   txReceipt?: TransactionReceipt;
 
+  /** The type of transaction this txMeta represents. */
```

Comment:
[](url)
```suggestion
  /** The type of transaction such as `cancel` or `swap`. */
```

Reply:
Pushed the changes manually. After I pushed some changes from the previous comments, it's not accepting the suggestions via the button =/